generators/app/files/CONTRIBUTING.md

```diff
@@ -2,9 +2,123 @@
+|-----------|---------------------|
+| `pnx`     | `init_phoenix`      |
+| `logo`    | `logo_v2`           |
+| `linter`  | `ci_linter_task`    |
+| `sockets` | `setup_web_sockets` |
+| `#42`     | `switch_to_adga`    |
+
+### Long-Running & Integration Branches
+
+In the edge case that a series of features are tightly coupled, and it is impractical to decouple them:
+
+1. Create a long-running "integration branch" from `master`
+2. Branch sub-features from the integration branch
+  - *Do not commit directly into the long-running branch directly*
+3. Frequently rebase sub-branches from the integration branch, and the feature branch from `master`
+4. When the sub-feature is complete, pull request against integration
+5. When the all coupled features are complete and merged into the integration branch,
+   pull request the integration branch against `master`
+
+It may be helpful to create a checklist in the integration branch's description
+of the subtasks that belong to it.
+
+### Commit Messages
+
+Provide a summary message of fewer than 50 characters. If further detail is required,
+provide a bullet-point list (or paragraph) on the line(s) below.
+
 ## Pull Requests
 
-TODO: Explain pull request process.
+Before a branch gets merged into `master`, it *must* be reviewed by another team member.
+This has a number of benefits:
+
+- Exposes the team to new code
+  - Keeps everyone in the loop
+  - Improves the project's [bus factor](https://en.wikipedia.org/wiki/Bus_factor)
+- Catch bugs early on
+- Lowers the risk of feature or utility duplication
+- Style consistency
+
+### Titles
+
+When creating your Pull Request, provide a short and descriptive title (fewer than 50 characters).
+These will likely be close to the branch name. Format like a normal sentence.
+
+| Bad         | Good                           |
+|-------------|--------------------------------|
+| `Quick fix` | `Add User model typespecs`     |
+| `#42`       | `Reduce Sass deep nesting`     |
+| `BUTTONS`   | `Update contact buttons to v3` |
+
+#### Descriptions
+
+To help the reviewer understand what they're reviewing, provide a description of what
+the code is expected to do. Link to any material, documentation, or resources that
+may be helpful for context.
+
+As a preemptive measure, be sure to mention any gotchas, or areas that may need explanation.
+
+#### Link Issues
+
+In GitHub, when you mention an Issue in a comment or description, it creates a link to and from the PR.
+
+This provides the reviewer with context on what the PR is for, and lets the team know that the Issue is near to being complete.
+
+### Reviewing
+
+As the reviewer of the code, your responsibilities include (but are not limited to):
+
+- When possible, you have actually used the feature as a end-user
+- The code does what it is intended to
+- You *understand* the code (don't just skim it)
+- All of the tests pass
+- The test coverage is sufficient
+- The style is consistent with the rest of the project
```

Comment:
Yeah, for sure.

It would depend if there's a well defined style guide for the project. If something isn't covered by the style guide, the group should agree on style and add it to the guide/linter. However, some things aren't available to the linter specifically, like naming conventions (ex. NounAdjective or AdjectiveNoun), patterns, architecture, and so on.

I'm happy to take out the line, but that's my case for it :wink: